frontend/src/pages/Resources/Resources.js

```diff
@@ -34,9 +40,31 @@ export default function Resources() {
             <section className="main-content">
               <section className="resource-list">
                 <SearchForm />
+                <div className="search-suggestions">
+                  <h4 class="header-description">Filters:</h4>
+                  {filters.map((filter, filterIdx) => {
+                    return (
+                      <p
+                        key={filterIdx}
```

Comment:
Rather use the installed nanoid package for the `key` prop instead of the `filterIdx`